### DISCUSSION TITLE
CallBack on open or will open

### DISCUSSION THREAD

**Discussion by abdullahIsa** [enesozturk/react-native-hold-menu#58] [Ideas]
Hello, thanks for this but currently if the keyboard is open and I open the menu then it stays behind the keyboard, would be great if a callback function can be added for instance a "willOpenMenu", thanks very much.

**Comment by kesha-antonov** [comment]
Maybe this is fix for this usecase: https://github.com/enesozturk/react-native-hold-menu/pull/84
